bonsai/sdk/README.md

```diff
@@ -101,8 +102,8 @@ fn run_stark2snark(session_id: String) -> Result<()> {
                 continue;
             }
             "SUCCEEDED" => {
-                let snark_receipt = res.output;
-                eprintln!("Snark proof!: {snark_receipt:?}");
+                let receipt_buf = client.download(&res.output.unwrap())?;
+                let snark_receipt: Receipt = bincode::deserialize(&receipt_buf)?;
```

Comment:
Is this here just to test that we can deserialize properly?

Reply:
The testing of the serialization happens in the bonsai integration test suite which is wrapping up final touches now.

Reply:
This seems like dead code, but maybe the diff is throwing me off.

Reply:
Yeah its not used because its just a README example.